### ISSUE TITLE
Chat Copilot: Unable to generate bot response. Details: Error: 500: => System.InvalidOperationException: Sequence contains no elements 

### ISSUE THREAD

**Issue by AnilGayakwad** [microsoft/semantic-kernel#2520]
**Describe the bug**

**To Reproduce**
1. On semantic-kernel Chat Copilot
2. Connected to NextJs application Local plugin 
3. Interaction with Chat error's out
4. Some work with other sample plugins of this repo 

**Expected behavior**
Getting error 

Unable to generate bot response. Details: Error: 500: => System.InvalidOperationException: Sequence contains no elements at System.Linq.ThrowHelper.ThrowNoElementsException() at System.Linq.Enumerable.First[TSource](IEnumerable`1 source) at Microsoft.SemanticKernel.Skills.OpenAPI.OpenApi.OpenApiDocumentParser.ExtractRestApiOperations(OpenApiDocument document) at Microsoft.SemanticKernel.Skills.OpenAPI.OpenApi.OpenApiDocumentParser.ParseAsync(Stream stream, Boolean ignoreNonCompliantErrors, CancellationToken cancellationToken) at Microsoft.SemanticKernel.KernelOpenApiExtensions.RegisterOpenApiSkillAsync(IKernel kernel, Stream documentStream, String skillName, OpenApiSkillExecutionParameters executionParameters, CancellationToken cancellationToken) at Microsoft.SemanticKernel.KernelOpenApiExtensions.ImportOpenApiSkillFromUrlAsync(IKernel kernel, String skillName, Uri url, OpenApiSkillExecutionParameters executionParameters, CancellationToken cancellationToken) ...

**Screenshots**
Error page
![image](https://github.com/microsoft/semantic-kernel/assets/9487037/4b0e089b-c232-465c-828e-999414a04f0c)

![image](https://github.com/microsoft/semantic-kernel/assets/9487037/fe49c412-f01e-46cc-a400-70022a679dcc)

Swagger.Json
![image](https://github.com/microsoft/semantic-kernel/assets/9487037/91cb1c5b-124b-42c7-bf28-2ee958918019)

ai-plugin.json
![image](https://github.com/microsoft/semantic-kernel/assets/9487037/070879ef-7f59-4c99-96a6-78b7aecdd821)

**Platform**
 - OS: Window
 - IDE:  VS Code
 - Language: Javascript, Node Js,  NextJs, React

**Comment by AnilGayakwad** [comment]
Hi, 
Could someone assist me with this?

**Comment by matthewbolanos** [comment]
Thanks for raising this. Are you able to share the OpenAPI schema for your plugin so we can investigate further?

**Comment by gitri-ms** [comment]
I see the OpenAPI schema above (swagger.json). I'll take a look at this today.

**Comment by gitri-ms** [comment]
@AnilGayakwad could you confirm what version of the `Microsoft.SemanticKernel.Skills.OpenAPI` package you have? You can find that in `CopilotChatWebApi.csproj`. (This appears to me that it could be a duplicate of #1546, so I encourage you to make sure you are on the latest version.)

That said, we don't yet support relative/empty URLs in OpenAPI specs. That work is tracked by #1782. The workaround is to fill out the `servers` property in the swagger with an absolute URL.

@SergeyMenshykh @nacharya1 This seems like it might be a recurring issue for people trying to import OpenAI plugins, since the url is defined in `ai-plugin.json` but not in the swagger/OpenAPI spec. We might want to consider increasing the priority of this feature work, as being able to test plugins is becoming one of the flagship features in Chat Copilot.

**Comment by AnilGayakwad** [comment]
Thank you @gitri-ms, 
I managed to get it running. As you highlighted, the issue was related to URLs in the OpenAPI specs.
and I am using "0.18.230725.3-preview+29ecb71" version for `Microsoft.SemanticKernel.Skills.OpenAPI`. 

For others' reference, this is what the updated OpenApi spec looks like.
Swager.json 
![image](https://github.com/microsoft/semantic-kernel/assets/9487037/e7b4771b-b758-4025-9088-eafd48387921)

**Comment by SergeyMenshykh** [comment]
Hi @AnilGayakwad, I'm glad that the solution proposed by @gitri-ms worked.

As @gitri-ms mentioned, we don't yet support relative URLs. However, we have stopped failing in case no 'servers' attribute is provided or it's empty since SK 0.19.230804.2-preview. If it's not convenient to include the URL in the OpenAPI document, it can be supplied through the 'server-url' context variable. You can find more information about this in the [Example](https://github.com/microsoft/semantic-kernel/blob/a13690ad4b12a97676803bdb5405644966ab0012/dotnet/samples/KernelSyntaxExamples/Example22_OpenApiSkill_AzureKeyVault.cs#L55).

These are the options you can consider for your plugin until relative URLs are supported by OpenAPI plugins.

**Comment by gitri-ms** [comment]
Glad we could get you unblocked. I'm going to close this issue, but if you run into any other problems, feel free to reactivate it or open a new one.
